### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by dearchap**

### CODE REVIEW

File: site/content/en/guides/_index.md [hunk 1]
```diff
@@ -7,5 +7,5 @@ menu:
   main:
     weight: 20
 description: >
-    Reference for Kustomize CLI commands
+    Reference for Kustomize usage and best practices
```
Comment (Shell32-Natsu) [inline site/content/en/guides/_index.md:10]: @monopole Can we call this *best practices*?

Comment (monopole) [reply site/content/en/guides/_index.md:10]: yes, in the context of kustomize.
it goes here
https://kubernetes-sigs.github.io/kustomize/guides/

@dearchap please consider providing larger prs to improve the docs, thanks!

**Pull request merged**